clap-v3-utils/src/nonce.rs

```diff
@@ -23,7 +23,7 @@ fn nonce_arg<'a>() -> Arg<'a> {
         .long(NONCE_ARG.long)
         .takes_value(true)
         .value_name("PUBKEY")
-        .requires(BLOCKHASH_ARG.name)
+        .requires(NONCE_AUTHORITY_ARG.name)
```

Comment:
this shouldn't be strictly required.  typical behavior in the cli tools is that any "authority" type signer, that is not explicitly specified, falls back to the `DefaultSigner` (either the `CliConfig` keypair or argument to `--keypair KEYPAIR`)

Reply:
Ah yes, of course the nonce authority can be from `CliConfig` or a separate keypair argument. Let's remove this requirement. 

It does seem like the token-cli does require a specific nonce-authority argument with the nonce argument, but perhaps I need to look at it a bit more closely.